src/System.Buffers/tests/project.json

```diff
@@ -1,17 +1,19 @@
 ﻿{
   "name": "System.Buffers.Tests",
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1-*",
+    "NETStandard.Library": "1.0.0-rc2-23728",
     "src": {
           "type": "build",
           "version": "0.1.0-*"
         },
     "System.Slices": "0.1.0-*",
-    "xunit": "2.1.0"
+    "xunit": "2.1.0",
+    "dotnet-test-xunit": "1.0.0-dev-45337-57"
   },
   "frameworks": {
-    "dotnet5.6": {
+    "dnxcore50": {
```

Comment:
Ah get it. I assume this is temporary though. Once our test dependencies move to std library, we would stop using dnxcore50.

Anyway, the change looks good.